skimage/exposure/_adapthist.py

```diff
@@ -119,8 +117,7 @@ def _clahe(image, kernel_size, clip_limit, nbins):
 
     The number of "effective" graylevels in the output image is set by `nbins`;
     selecting a small value (eg. 128) speeds up processing and still produce
-    an output image of good quality. The output image will have the same
-    minimum and maximum value as the input image. A clip limit smaller than 1
+    an output image of good quality. A clip limit of 0 or larger equal 1
```

Comment:
Is "a clip limit either == 0 or >=1" what is intended?

Reply:
For CLAHE, the clip value would be in the open interval (0, 1). Clip values >=1 essentially disable clipping and therefore simply perform AHE. A clip value of 0 does not really make sense, so the implementation falls back to AHE in this case.

Reply:
Thanks for the very clear response!